backend-flask/README.txt

```diff
@@ -0,0 +1,68 @@
+INSTALL REQUIREMENTS
+You can install flask and other requirements such as RESTfulapi for flask by simply running this script:
+
+$ pip install -r requirements.txt
+
+-------------------------
+VENV
+To install venv enter the following command:
+
+$ python3 -m venv venv
```

Comment:
I would add a note here that when installing `venv`, use the version of python that support venv (i.e. 3.3+), as `python3` isn't consistent across all systems. For example, `py` refers to Python 3.7.1 for me while `python` refers to Python 2.x.

It's just a minor wording thing; up to you if you want to change it.